lib/models/device.coffee

```diff
@@ -2099,6 +2099,26 @@ getDeviceModel = (deps, opts) ->
 
 		return timeSince(lce)
 
+	###*
+	# @summary Get the composite version number of the OS running on a device
+	# @name getOsVersion
+	# @public
+	# @function
+	# @memberof balena.models.device
+	#
+	# @param {Object} device - A device object
+	# @returns {String}
+	#
+	# @example
+	# balena.models.device.get('7cf02a6').then(function(device) {
+	# 	console.log(device.os_version); // => 'balenaOS 2.26.0+rev1'
+	# 	console.log(device.os_variant); // => 'prod'
+	# 	balena.models.device.getOsVersion(device); // => '2.26.0+rev1.prod'
+	# })
+	###
+	exports.getOsVersion = (device) ->
```

Comment:
I couldn't really decide a name for this, so I'm looking forward for suggestions as well.
The name should make clear that this is the complete os release version number but also not confuse the user about the `balenaOS` prefix.
The other name that I had in mind was `getOsRelease`, trying to describe that this semver is from the complete OS release version, but I couldn't convince my self to use it.

Reply:
I think `getOsRelease` might be even more confusing (is that a release ID, a version?). I think `getOsVersion` is fine if we want to keep it short, maybe the variant is the only part that would be unexpected as part of the return value, but it is documented that it is included so it shouldn't be an issue. We can potentially add a second `options` param to specify what to include/exclude if necessary in the future.